substra/sdk/backends/local/backend.py

```diff
@@ -252,22 +262,13 @@ def __execute_compute_plan(
         return compute_plan
 
     def __format_for_leaderboard(self, testtuple):
```

Comment:
This is a bit too late, but IMHO we don't need to support the leaderboard function in the debugging mode. As it is already done, we can keep it like this :)